acceptance/performance/facts/fact_load.rb

```diff
@@ -2,7 +2,8 @@
   clear_and_restart_puppetdb(database)
 
   step "run benchmark" do
-    on(database, "#{LeinCommandPrefix} lein run benchmark -F acceptance/benchmark/facts/ -n 1000 -c acceptance/benchmark/config.ini -N 5 --rand-perc 10")
+    sample_data_path = "resources/puppetlabs/puppetdb/benchmark"
+    on(database, "#{LeinCommandPrefix} lein run benchmark -F #{sample_data_path}/samples/facts/ -n 1000 -c #{sample_data_path}/config.ini -N 5 --rand-perc 10")
```

Comment:
I'm not sure I understand how this works. For the performance tests we're using this sample data and it's looking in resources/puppetlabs/puppetdb/benchmark. Does this mean we're perf testing against source, not a package? Not a problem, just wondering how that works /cc @kbarber